### DISCUSSION TITLE
connecting to production server

### DISCUSSION THREAD

**Discussion by yongra** [tl-its-umich-edu/my-learning-analytics#1334] [General]
Hello,

If I set up the MyLA server to the production Cavas server, could there be any problem with the Canvas if there are any setup issues with the MyLA server? 

For that reason, I am wanted to link the MyLA server to the test server, but it does not seem to work. Any input will be appreciated.

Ra

**Comment by zqian** [comment]
@yongra please follow the [instructions for setting up LTI in dev/test environment](https://github.com/tl-its-umich-edu/my-learning-analytics/wiki/Dev:-Configuration-for-LTI-1.3-Development) first, before deploying to Prod Canvas.

**Reply by yongra** [reply]
The test environment section is asking the server to be set up with ngrok. My server is open to the internet. So, I don't think I need to set up with ngrok, or do I?

What ports do MyLA require to be opened? TCP 443 is opened. Does it require any other ports?

**Comment by yongra** [comment]
In order for this to work, do I need to download and install the SQL file from Google drive and load the database? Even though I am not using the data in the file from Google drive.
